exist-core/src/test/java/org/exist/backup/XMLDBRestoreTest.java

```diff
@@ -290,57 +298,55 @@ private static void restoreBackup(final XmldbURI uri, final Path backup, @Nullab
-                        "    <resource type=\"BinaryResource\" name=\"doc4.html\" owner=\"admin\" group=\"dba\" mode=\"644\" created=\"2019-05-15T15:58:49.618+04:00\" modified=\"2019-05-15T15:58:49.618+04:00\" filename=\"doc4.html\" mimetype=\"text/xhtml\">\n" +
-                        "        <acl entries=\"0\" version=\"1\"/>\n" +
-                        "    </resource>\n" +
-                        "    <resource type=\"BinaryResource\" name=\"doc5.html\" owner=\"admin\" group=\"dba\" mode=\"644\" created=\"2019-05-15T15:58:49.618+04:00\" modified=\"2019-05-15T15:58:49.618+04:00\" filename=\"doc5.html\" mimetype=\"text/html\">\n" +
-                        "        <acl entries=\"0\" version=\"1\"/>\n" +
-                        "    </resource>\n" +
-                        "</collection>";
-
-        final String doc1 = "<doc1/>";
-        final String doc2 = "<doc2/>";
-        final String doc3 = "<doc3/>";
-        final String doc4 = "<html><body><hr/></body></html>";
-        final String doc5 = "<html><body><hr></body></html>";
+        final StringBuilder col1Contents = new StringBuilder();
+        col1Contents.append("<collection xmlns=\"http://exist.sourceforge.net/NS/exist\" name=\"/db/").append(COLLECTION1_NAME).append("\" owner=\"admin\" group=\"dba\" mode=\"755\" created=\"2019-05-15T15:58:39.385+04:00\" deduplicate-blobs=\"false\" version=\"2\">\n");
+        col1Contents.append("    <acl entries=\"0\" version=\"1\"/>\n");
+        for (final DocInfo backupDocInfo : BACKUP_DOCS) {
+            col1Contents.append("    <resource type=\"").append(backupDocInfo.type == MimeType.XML ? "XMLResource" : "BinaryResource").append("\" name=\"").append(backupDocInfo.name).append("\" owner=\"admin\" group=\"dba\" mode=\"644\" created=\"2019-05-15T15:58:48.638+04:00\" modified=\"2019-05-15T15:58:48.638+04:00\" filename=\"").append(backupDocInfo.name).append(backupDocInfo.mediaType != null ? "\" mimetype=\"" + backupDocInfo.mediaType + "\">\n" : "\">\n");
+            col1Contents.append("        <acl entries=\"0\" version=\"1\"/>\n");
+            col1Contents.append("    </resource>\n");
+        }
+        col1Contents.append("</collection>");
 
         final Path dbContentsFile = Files.write(db.resolve(BackupDescriptor.COLLECTION_DESCRIPTOR), dbContents.getBytes(UTF_8));
-        final Path col1ContentsFile = Files.write(col1.resolve(BackupDescriptor.COLLECTION_DESCRIPTOR), col1Contents.getBytes(UTF_8));
-        Files.write(col1.resolve("doc1.xml"), doc1.getBytes(UTF_8));
-        Files.write(col1.resolve("doc2.xml"), doc2.getBytes(UTF_8));
-        Files.write(col1.resolve("doc3.xml"), doc3.getBytes(UTF_8));
-        Files.write(col1.resolve("doc4.html"), doc4.getBytes(UTF_8));
-        Files.write(col1.resolve("doc5.html"), doc5.getBytes(UTF_8));
+        final Path col1ContentsFile = Files.write(col1.resolve(BackupDescriptor.COLLECTION_DESCRIPTOR), col1Contents.toString().getBytes(UTF_8));
```

Comment:
I guess the local variable `col1ContentsFile`can be dropped as it is not used by itself (keeping the write operation)